documentation/src/main/wiki/ActionAdapter.md

```diff
@@ -0,0 +1,16 @@
+Action Adapter is Component of a system, that mediate communication between Knot.x (ActionKnot) and external 
+Services that are responsible for handling form submitting.
+
+Action Adapter is a project custom part of a system, however Knot.x comes with default `HttpActionAdapter`.
```

Comment:
Action Adapter is an extension of Knot.x. By delivering custom implementations of Action Adapters, that meet your project requirements, it allows you to adopt request from Knot.x into request understandable by endpoint service, and adopts responses from that service into unified message understandable by Knot.x.
For you convenience, Knot.x is shipped with a default Action Adapter called `HttpActionAdapter`.